src/viewer.js

```diff
@@ -2735,42 +2735,54 @@ function onCanvasPinch( event ) {
     return false;
 }
 
+var lastScroll = new Date().getTime();
 function onCanvasScroll( event ) {
     var gestureSettings,
-        factor;
-
-    if ( !event.preventDefaultAction && this.viewport ) {
-        gestureSettings = this.gestureSettingsByDeviceType( event.pointerType );
-        if ( gestureSettings.scrollToZoom ) {
-            factor = Math.pow( this.zoomPerScroll, event.scroll );
-            this.viewport.zoomBy(
-                factor,
-                this.viewport.pointFromPixel( event.position, true )
-            );
-            this.viewport.applyConstraints();
+        factor,
+        thisScroll,
+        deltaScroll;
+
+    /* Certain scroll devices fire the scroll event way too fast so we are injecting a simple adjustment to keep things
+     * partially normalized. If we have already fired an event within the last 'minScrollDelta' milliseconds we skip
+     * this one and wait for the next event. */
+    thisScroll = new Date().getTime();
+    deltaScroll = thisScroll - lastScroll;
```

Comment:
These variables should be named appropriate to time; otherwise it's confusing with the distance scrolled. For instance: `deltaScrollTime = now - this._lastScrollTime`.